### PR TITLE
Adding integration tests for spatial aggregation

### PR DESCRIPTION
**Pull request by mmcentre**
Added two integration tests. The first one is testing aggregating of climate (gridMET) data over counties. The second is testing aggregating PM25 with components over ZCTA areas.

Added test are automatically generated, but require a manual validation.
Testing pipelines are created by [gen_dbt_cwl](https://github.com/NSAPH-Data-Platform/nsaph-core-platform/blob/develop/src/python/nsaph/dbt/gen_dbt_cwl.py) utility given the pipeline being tested. Once the pipeline is run once and have created table(s) in Dorieh, the [create_test](https://github.com/NSAPH-Data-Platform/nsaph-core-platform/blob/develop/src/python/nsaph/dbt/create_test.py) utility generates a test script testing that table fingerprints do not change.

### CODE REVIEW

File: src/cwl/pm25_yearly_download.cwl [hunk 1]
```diff
@@ -36,6 +36,40 @@ doc: |
   CSV files. This is a wrapper around actual aggregation of
   one file allowing to scatter (parallelize) the aggregation
   over years.
+  
+  The output of the workflow are gzipped CSV files containing
```
Comment (audiracmichelle) [inline src/cwl/pm25_yearly_download.cwl:40]: Hi Misha, can the output have a parquet option?

**Review by audiracmichelle — APPROVED** [review]
Hi Misha, it's great to have the database.ini option separated from gzip csv version.
currently the tests pull the output from the DB, correct?

File: src/cwl/pm25_yearly_download.cwl (hunk 1 above)
Comment (mmcentre) [reply src/cwl/pm25_yearly_download.cwl:40]: Hi Michelle, I think it is quite trivial to do but it needs to be done and tested. Do you want me to add this option now or put this in a queue of tasks?
Note, that we can also add outputting as FST or HDF5, I have implemented it on Mahmood requests long time ago but have never enabled it as it seemed that nobody has needed these formats.

Comment (audiracmichelle) [reply src/cwl/pm25_yearly_download.cwl:40]: we can leave it in the queue for now. Thanks!

**Pull request merged**